Readme.md

```diff
@@ -115,6 +115,9 @@ var nightmare = Nightmare({
 });
 ```
 
+##### ['web-preferences'].preload
+Specifies a file to add to preload.  The generated preload file is written to Electron's `userData` path.
```

Comment:
Sure, I think I still like the parentheses around the second sentence though to indicate that it's a separate note of lesser importance.